docs/en_US/sqls/functions/string_functions.md

```diff
@@ -188,4 +188,20 @@ Removes all leading and trailing whitespace (tabs and spaces) from the provided
 upper(col)
 ```
 
-Returns the uppercase version of the given string.
\ No newline at end of file
+Returns the uppercase version of the given string.
```

Comment:
Suggest to document this in transform_functions. Also, use uppercase in the title like ##HEX2DEC

Also, need to point out that the hex here must have `0x` prefix.